docs/pages/web/helpbutton.js

```diff
@@ -0,0 +1,265 @@
+        <MainSection.Subsection description="Be sure to localize `text`, `accessibilityLabel`, `accessibilityPopoverLabel`, `link.text` and `link.accessibilityLabel` within HelpButton. Note that localization can lengthen text by 20 to 30 percent." />
+      </MainSection>
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Link"
+          description="HelpButton's popover content can contain a link to additional information. If needed, this interaction can be paired with [OnLinkNavigationProvider](/web/utilities/onlinknavigationprovider)."
+        >
+          <MainSection.Card
+            cardSize="md"
+            defaultCode={`
+<Flex height="100%" justifyContent="center" alignItems="center" gap={1}>
+  <Text>This is Gestalt</Text>
+  <HelpButton
+    accessibilityLabel="Click to learn more about gestalt"
+    accessibilityPopoverLabel="Expanded information about Gestalt"
+    link={{
+      href: 'https://gestalt.pinterest.systems/',
+      text: 'Visit our portal',
+      accessibilityLabel: 'Visit Gestalt portal',
+    }}
+    text="Gestalt is Pinterest's design system."
+  />
+</Flex>
+        `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            sandpackExample={<SandpackExample code={withLink} name="Link component" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Text"
+          description="The `text` prop accepts either a string or [Text](/web/text). Use a string for simple text without any visual style. HelpButton will handle the message style and adherence to design guidelines. If a message with more complex style is required, such as bold text or inline links, use Text to wrap your message with any additional Text contained within."
+        >
+          <MainSection.Card
+            cardSize="md"
+            defaultCode={`
+<Flex height="100%" justifyContent="center" alignItems="center" gap={1}>
+  <Text>This is Gestalt</Text>
+  <HelpButton
+    accessibilityLabel="Click to learn more about gestalt"
+    accessibilityPopoverLabel="Expanded information about Gestalt"
+    link={{
+      href: 'https://gestalt.pinterest.systems/',
+      text: 'Visit our portal',
+      accessibilityLabel: 'Visit Gestalt portal',
+    }}
+    text={<Text><Text weight="bold" inline>Gestalt</Text> is Pinterest's design system.</Text>}
+  />
+</Flex>
+        `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            sandpackExample={<SandpackExample code={withText} name="Text component" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Scrollable Containers"
+          description="Some cases you will need use the HelpButton inner a [ScrollBoundaryContainer](/web/utilities/scrollboundarycontainer) implementation, which can start some not expected behaviors. To avoid this mismatches use `isWithinScrollContainer` as `true`."
```

Comment:
In some cases, you will need to use HelpButton within [ScrollBoundaryContainer](/web/utilities/scrollboundarycontainer). To avoid unexpected behaviors  set `isWithinScrollContainer` to "true"."

Reply:
Done